sql/size_utils.sql

```diff
@@ -92,7 +92,9 @@ $BODY$
             0::BIGINT AS compressed_toast_size,
             0::BIGINT AS compressed_heap_size
         FROM
-            _timescaledb_catalog.hypertable
+            _timescaledb_catalog.hypertable ht
+            INNER JOIN pg_class c ON relname = ht.table_name
+            AND c.relnamespace IN (SELECT oid FROM pg_namespace WHERE nspname = schema_name_in)
             JOIN LATERAL _timescaledb_internal.relation_size(
                 format('%I.%I', schema_name, table_name)::regclass) AS relsize ON TRUE
```

Comment:
```suggestion
            _timescaledb_catalog.hypertable ht
            INNER JOIN pg_class c ON c.relname = ht.table_name AND c.relkind = 'r'
            INNER JOIN pg_namespace n ON n.oid = c.relnamespace AND n.nspname = ht.schema_name
            JOIN LATERAL _timescaledb_internal.relation_size(c.oid) AS relsize ON TRUE
```
And something similar should also be done in the `_timescaledb_internal.hypertable_chunk_local_size` view for compressed and uncompressed chunks.

Reply:
Since we're joining the `pg_class` now there is no need to cast the schema and table to a regclass in the `relation_size` function call, just use the `pg_class.oid` as parameter.

Reply:
Added similar changes in `_timescaledb_internal.hypertable_chunk_local_size` to to join timescale catalog table with `pg_class` and also updated `relation_size` function to use `oid` from `pg_class`.